rb/lib/selenium/webdriver/common/error.rb

```diff
@@ -60,57 +70,58 @@ class StaleElementReferenceError < WebDriverError; end # 10
       class ElementNotVisibleError < WebDriverError; end # 11
 
       #
-      # Raised when an interaction could not be performed because the element
-      # is in an invalid state (e.g. attempting to click a disabled element).
+      # A command could not be completed because the element is in an
+      # invalid state, e.g. attempting to click an element that is no
```

Comment:
It's been decided! 

`The target element is in an invalid state, rendering it impossible to interact with, for example if you click a disabled element.`